src/style.css

```diff
@@ -1,3 +1,75 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Roboto:wght@400;700&display=swap");
```

Comment:
you need to use Roboto 500 font weight 
![image](https://github.com/mate-academy/layout_moyo-header/assets/96265844/33b98728-6c3d-400b-a0c5-f03e0803459f)